src/python/bot/tasks/task_creation.py

```diff
@@ -271,12 +273,19 @@ def request_bisection(testcase):
   if not target:
     return
 
-  _make_bisection_request(pubsub_topic, testcase, target, 'regressed')
-  _make_bisection_request(pubsub_topic, testcase, target, 'fixed')
+  # Only make 1 request of each type per testcase.
```

Comment:
What happens if that request fails ? Should we check pubsub_client.publish return value in _make_bisection_request

Reply:
Only way that can fail will be via an exception.